.pre-commit-config.yaml

```
@@ -2,4 +2,27 @@ repos:
 -   repo: https://github.com/psf/black
     rev: 19.10b0
     hooks:
-    - id: black
\ No newline at end of file
+    - id: black
+-   repo: https://github.com/asottile/seed-isort-config
+    rev: v2.1.1
+    hooks:
+    -   id: seed-isort-config
+-   repo: https://github.com/timothycrosley/isort
+    rev: 4.3.21
+    hooks:
+    -   id: isort
+-   repo: https://github.com/Yelp/detect-secrets
+    rev: v0.14.2
+    hooks:
+    -   id: detect-secrets
+        args: ['--baseline', '.secrets.baseline']
+-   repo: https://github.com/Lucas-C/pre-commit-hooks-nodejs
+    rev: v1.1.1
+    hooks:
+    -   id: markdown-toc
+        files: ^README.md$
+-   repo: https://github.com/asottile/blacken-docs
+    rev: v1.7.0
```

`blacken-docs` is in v1.8.0
```suggestion
    rev: v1.8.0
```